### DISCUSSION TITLE
Automatic Half Scaling

### DISCUSSION THREAD

**Discussion by ZacWarham** [mealie-recipes/mealie#3607] [Ideas]
I like being able to use the scaling option although as mentioned in #2613; the only way currently to create a smaller scale is clicking on the servings and using the edit scale pop-up. Could the button group be edited to the following so that the decrease button halves the pop-up?

This does introduce the issue of the button text being inaccurate when going below zero as it is no longer taking away a full amount but is halving it.

> frontend/components/Domain/Recipe/RecipeScaleEditButton.vue

```
<BaseButtonGroup
        v-if="editScale"
        class="pl-2"
        :large="false"
        :buttons="[
          {
            icon: $globals.icons.minus,
            text: $t('recipe.decrease-scale-label'),
            event: 'decrement',
          },
          {
            icon: $globals.icons.createAlt,
            text: $t('recipe.increase-scale-label'),
            event: 'increment',
          },
        ]"
        @decrement="scale > 1 ? scale-- : scale = scale / 2"
        @increment="scale > 1 ? scale++ : scale = scale * 2"
      />
```

**Comment by ChipSkylark37** [comment]
I would like this as well. It could mess up some of the ingredients, as say a recipe that would call for '1 egg'... how can you easily scale that down below 1 and have any accuracy? But that could happen for anything - I do agree that I'd like to be able to not have to manually adjust when its < 1  scale and also be able to save that scale as the default unless I change it.
